bin/openstack-install

```diff
@@ -193,18 +193,6 @@ if __name__ == '__main__':
             print('Only supports killing a single install for now.')
             sys.exit(1)
 
-    if os.path.isfile(os.path.join(cfg.cfg_path, 'installed')):
-        msg = ("\n\nError:\n\n"
-               "Previous installation detected. Did you mean to run "
-               "openstack-status instead? \n"
-               "If attempting to re-install please remove "
-               "~/.cloud-install or move it to a different \n"
-               "directory name (eg ~/cloud-install-old). "
-               "Otherwise you may wish to uninstall with:\n\n"
-               "    $ sudo openstack-install -u\n\n")
```

Comment:
I thought about this a bit. I think maybe we need additional install states that could be more clear at what point an install actually failed. That way we don't need to actually blow the whole thing away if a container was never created due to an error etc.

Lemme think on this more, if you've got any ideas feel free